src/layout/strategy/container.js

```diff
@@ -3,10 +3,12 @@ export default function() {
         containerHeight = 0;
 
     var container = function(point) {
-        return !(point.x < 0 || point.y < 0 ||
+        // If the bounds haven't been defined, then don't enforce
```

Comment:
I think, in this instance, evaluating against `null` would be more sensible, since `0` would probably be a valid value (if it was for a case of explicitly always failing). I've made the change.